cirq-google/cirq_google/engine/engine_sampler.py

```diff
@@ -139,3 +139,106 @@ def get_engine_sampler(
         )
     gate_set = gate_sets.NAMED_GATESETS[gate_set_name]
     return engine.get_engine(project_id).sampler(processor_id=processor_id, gate_set=gate_set)
+
+
+def get_device_sampler(
+    project_id: Optional[str] = None, processor_id: Optional[str] = None
+) -> Tuple[
+    Tuple[Union[cirq.Device], int],
+    Union['cirq_google.PhasedFSimEngineSimulator', 'cirq_google.QuantumEngineSampler'],
+    bool,
+]:
+    """Authenticates on Google Cloud, can return a Device and Simulator.
+
+    This uses the environment variable GOOGLE_CLOUD_PROJECT for the Engine
+    project_id, unless set explicitly.
```

Comment:
nit: Can remove this since it is redundant with the `project_id` description below.